Connected System Template (CST) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/src/main/resources/appian-plugin.xml

```diff
@@ -9,9 +9,14 @@
 
   <connected-system-template key="GoogleDriveSampleConnectedSystem" name="GoogleDriveSampleConnectedSystem"
                              class="com.mycorp.helloworld.templates.GoogleDriveSampleConnectedSystemTemplate">
```

Comment:
should our examples be back compatible as well.
before your change, if someone installed this sample template and created some connected-systems
then saw these new changes and wanted them as well, I dont think we want to break their existing configurations.

we should change the key and update the TemplateId versions to #2